src/kernel/environment.c

```diff
@@ -1,23 +1,24 @@
 #include <environment.h>
 #include <drivers/kinetis/uart.h>
+#include <drivers/kinetis/i2c.h>
 
 // UART 0 CONFIGURATION (OpenSDA UART <-> USB)
 uart_config uart0_conf = {
     .uart_base = UART0,
-    .rx_pcr = &(PORTB->PCR[16]),
-    .tx_pcr = &(PORTB->PCR[17]),
+    .rx_pcr = &(PORTA->PCR[1]),
+    .tx_pcr = &(PORTA->PCR[2]),
```

Comment:
I feel like we should have left these hunks out, and include them only on a demo branch, since during normal debugging we're going to want the multiplexed OpenSDA + UART streams.